crates/agent-sql/src/id.rs

```diff
@@ -1,3 +1,4 @@
+use hex::FromHexError;
```

Comment:
I like the idea of making it clear what exports come from where... but I'm not clear what you're actually suggesting here?

Reply:
tactically, in this case it means using `hex::FromHexError` in usage below rather than `use hex::FromHexError` here and later `FromHexError` below.

Reply:
Ahh, okay! I've always hesitated for about 0.5s every time I go to `use` vs fully qualify the path to an outside type, so I'm happy to have a heuristic to follow.

That being said... I'm actually still not entirely sure what that heuristic is. It can't be "always fully qualify types that come from external dependencies", as that would get really wordy. Is it more like "always fully qualify types _that are part of an exposed interface_ if they come from external dependencies"?

Reply:
I don't know that I have a more precise definition, and certainly don't want to bike-shed over it, but simply wanted to raise it for you to think about. The heuristic I try and use is "what's going to be easier to quickly comprehend for the _reader_ of this module (or a few lines of diff thereof)?". That's the North Star -- because code we write once will be read many many times -- so if we try and stay true to that we'll be okay.

Reply:
That makes sense!